its/ruling/src/test/resources/expected/python-S3827.json

```diff
@@ -12,6 +12,11 @@
 'project:numpy-1.16.4/numpy/linalg/lapack_lite/clapack_scrub.py':[
 306,
 ],
+'project:numpy-1.16.4/numpy/matlib.py':[
```

Comment:
those are FPs because the imported module (i.e. `numpy`) does some additional manipulation with `__all__` property.
I'm willing to accept this FP, knowing that it will be treated in [SONARPY-547](https://jira.sonarsource.com/browse/SONARPY-547)